README.md

```diff
@@ -1,10 +1,139 @@
 # GenZ
-## Requirements
 
-- [Go >1.20](https://go.dev/doc/install)
-## Try it out
-Explore built-in `examples`, clone repo, and run
```

Comment:
I would have kept this (invite the user to browse our examples) after the ## Getting started example